File: roles/comment-writer/tests/fixtures/obvious-en/input.ts
// validate the order parameters
function validateOrder(order: Order): boolean {
  // loop over the items
  for (const item of order.items) {
    // check the stock
    if (item.stock < item.qty) {
      return false;
    }
  }
  return true;
}

// initialize the payment client
const client = new PaymentClient(config.payKey);

// compute the line total
function lineTotal(item: Item): number {
  return item.price * item.qty;
}

// compute the order total
function orderTotal(items: Item[]): number {
  return items.reduce((sum, item) => sum + lineTotal(item), 0);
}

// set the request timeout
function setRequestTimeout(seconds: number): void {
  config.timeout = seconds;
}

File: roles/comment-writer/tests/actual/obvious-cn/input.py
def validate_order(order):
    for item in order["items"]:
        if item["stock"] < item["qty"]:
            return False
    return True


client = PaymentClient(config["pay_key"])


def _line_total(item):
    return item["price"] * item["qty"]


def _order_total(items):
    return sum(_line_total(item) for item in items)


def set_timeout(seconds):
    config["timeout"] = seconds
